apps/dashboard/app/new/page.tsx

```diff
@@ -24,6 +24,8 @@ type Props = {
 
 
 export default async function (props: Props) {
+  const _auth = await getAuth(); 
```

Comment:
what's this for? it's not used it seems. why are we assigning it to a variable?

Reply:
https://github.com/unkeyed/unkey/pull/3083#discussion_r2033211740